README.md

```diff
@@ -128,8 +128,6 @@ out:
 
 The geographic location of the dataset. Required except for US and EU.
 
-`auto_create_table` isn't supported except for US and EU. And GCS bucket should be in same region when you use `gcs_bucket`.
```

Comment:
@potato2003 I think it's better to keep description "GCS bucket should be in same region when you use `gcs_bucket`".
Option name `auto_create_table` should be replaced to `auto_create_gcs_bucket`, but.
> You are subject to the following limitations when you load data into BigQuery from a Cloud Storage bucket:
> 
> If your dataset's location is set to a value other than US, the regional or multi-regional Cloud Storage bucket must be in the same region as the dataset.

https://cloud.google.com/bigquery/docs/loading-data-cloud-storage#limitations